### DISCUSSION TITLE
How does resizing images for training works?

### DISCUSSION THREAD

**Discussion by alierenaltindag** [ultralytics/ultralytics#1024] [Q&A]

**Comment by glenn-jocher** [comment]
@alierenaltindag hello! When training YOLOv8, images are resized to ensure a consistent input size for the neural network. This resizing is done while maintaining the aspect ratio of the original image to avoid distortion. Padding is added to the resized image if necessary to reach the desired network input size. This process allows the model to train on images of various dimensions while providing it with a uniform input shape. For more details on how this works and how you can adjust the settings, please refer to the Data Preprocessing section in our documentation. Happy training! 😊🚀
